Maddori.Apple/Maddori.Apple/Screen/AddFeedbackContent/AddFeedbackContentViewController.swift

```diff
@@ -153,17 +196,67 @@ final class AddFeedbackContentViewController: BaseViewController {
             $0.top.equalTo(feedbackContentLabel.snp.bottom).offset(SizeLiteral.labelComponentPadding)
             $0.leading.equalTo(addFeedbackContentView.snp.leading).inset(SizeLiteral.leadingTrailingPadding)
             $0.trailing.equalTo(addFeedbackContentView.snp.trailing).inset(SizeLiteral.leadingTrailingPadding)
-            $0.height.greaterThanOrEqualTo(150)
+            $0.height.equalTo(150)
+        }
+        
+        addFeedbackContentView.addSubview(feedbackStartSwitch)
+        feedbackStartSwitch.snp.makeConstraints {
+            $0.top.equalTo(feedbackContentTextView.snp.bottom).offset(SizeLiteral.componentIntervalPadding)
+            $0.trailing.equalTo(addFeedbackContentView.snp.trailing).inset(SizeLiteral.leadingTrailingPadding)
+            $0.width.equalTo(51)
+            $0.height.equalTo(31)
+        }
+        
+        addFeedbackContentView.addSubview(feedbackStartLabel)
+        feedbackStartLabel.snp.makeConstraints {
+            $0.centerY.equalTo(feedbackStartSwitch.snp.centerY)
+            $0.leading.equalTo(addFeedbackContentView.snp.leading).inset(SizeLiteral.leadingTrailingPadding)
         }
         
-        addFeedbackContentView.addSubview(feedbackDoneButton)
+        addFeedbackContentView.addSubview(feedbackStartTextViewLabel)
+        feedbackStartTextViewLabel.snp.makeConstraints {
+            $0.top.equalTo(feedbackStartSwitch.snp.bottom).offset(SizeLiteral.componentIntervalPadding)
+            $0.leading.equalTo(addFeedbackContentView.snp.leading).inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        addFeedbackContentView.addSubview(feedbackStartTextView)
+        feedbackStartTextView.snp.makeConstraints {
+            $0.top.equalTo(feedbackStartTextViewLabel.snp.bottom).offset(SizeLiteral.labelComponentPadding)
+            $0.leading.equalTo(addFeedbackContentView.snp.leading).inset(SizeLiteral.leadingTrailingPadding)
+            $0.trailing.equalTo(addFeedbackContentView.snp.trailing).inset(SizeLiteral.leadingTrailingPadding)
```

Comment:
p4;

```suggestion
            $0.leading.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
```
요기 2개도 수정할 수 있을 것 같습니다 !
다른 부분들은 굳이 코멘트 남기진 않겠습니다 ! 수정하면서 같이 부탁드릴게요 ^_______^

Reply:
[e4aab51](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/47/commits/e4aab5173c8b639771a8c4092987ea5ddf08fff0)
수정해서 반영했습니다 ! 감사합니다 :)